website_sale_product_pack/views/assets.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="assets_frontend" inherit_id="website_sale.assets_frontend">
```

Comment:
Isn't there specific assets for tests? Or at least load it on demo.

Reply:
Yes. Done